topics/chip-seq/tutorials/formation_of_super-structures_on_xi/tutorial.md

```diff
@@ -0,0 +1,718 @@
+>
+{: .hands_on}
+
+We can now concatenate the MACS2 outputs with the location of the peaks (concatenate the files and merge the overlapping regions) to obtain one BED file corresponding to the coordinates of the interesting regions to plot.
+
+> ### {% icon hands_on %} Hands-on: Prepare the peak coordinates
+> 
+> 1. **Concatenate two datasets into one dataset** {% icon tool %} with the following parameters
+>    - {% icon param-file %} *"Concatenate"*: output of **MACS2 callpeak** {% icon tool %} for `wt_CTCF_rep1`
+>    - {% icon param-file %} *"with"*: output of **MACS2 callpeak** {% icon tool %} for `wt_H3K4me3_rep1`
+> 2. **SortBED** {% icon tool %} with the following parameters
+>    - {% icon param-file %} *"Sort the following bed,bedgraph,gff,vcf file"*: output of **Concatenate** {% icon tool %}
+> 3. **MergeBED** {% icon tool %} with the following parameters
+>    - {% icon param-file %} *"Sort the following bed,bedgraph,gff,vcf file"*: output of **SortBED** {% icon tool %}
+>
+{: .hands_on}
+
+Now, plotting the peaks will involve using two tools:
+- **computeMatrix** {% icon tool %}: it computes the signal on given regions, using the `bigwig` coverage files from different samples.
+- **plotHeatmap** {% icon tool %}: it plots heatMap of the signals using the **computeMatrix** {% icon tool %} output.
+
+Optionally, we can also use **plotProfile** {% icon tool %} to create a profile plot using to **computeMatrix** {% icon tool %} output.
+
+> ### {% icon hands_on %} Hands-on: Plot the heatmap
+> 
+> 1. **computeMatrix** {% icon tool %} with the same parameters but:
+>    - *"Select regions"*: 
+>       - {% icon param-file %} *"Regions to plot"*: output of **MergeBED** {% icon tool %}
+>    - *"Sample order matters"*: `No`
+>       - {% icon param-files %} *"Score file"*: the 2 `bigwig` files generated by **bamCompare** {% icon tool %} and renamed
+>    - *"computeMatrix has two main output options"*: `reference-point`
+>       - *"The reference point for the plotting"*: `center of region`
+>       - *"Distance upstream of the start site of the regions defined in the region file"*: `3000`
+>       - *"Distance downstream of the end site of the given regions"*: `3000`
+> 2. **plotHeatmap** {% icon tool %} with the following parameters
+>    - {% icon param-file %} *"Matrix file from the computeMatrix tool"*: `Matrix` (output of **computeMatrix** {% icon tool %})
+>    - *"Show advanced options"*: `yes`
+>       - *"Reference point label"*: select the right label
+>       - *"Did you compute the matrix with more than one groups of regions?"*: `No, I used only one group`
+>           - *"Clustering algorithm"*: `Kmeans clustering`
+>           - *"Number of clusters to compute"*: `2`
+{: .hands_on}
+
+It should generate an heatmap similar to:
+
+![Output of plotHeatmap for 2 samples](../../images/formation_of_super-structures_on_xi/peak_heatmap_1.png "Scores around the peaks for wt_H3K4me3_rep1 and wt_input_rep1")
```

Comment:
The graphics could be explained better. Especially if someone sees this for the first time it is a bit difficult to interpret it.